model-mapper/src/Main.java

```diff
@@ -0,0 +1,58 @@
+import com.baeldung.model.User;
```

Comment:
we don't create new modules unless necessary and most certainly not in the root, so let's move the code to an existing module. I think a good candidate would be java-collections-conversions-2. do not create a new maven module, but rather use the existing one and create a package called modelmapper for example

Reply:
does this mean that the code should be put in the test package of the module?

Reply:
no, we use both. code that matters on the src and test related stuff (test classes and possible helpers on the test)

Reply:
Ok. i almost forget. the model-mapper.jar dependency should be added to pom?

Reply:
we use maven:
https://mvnrepository.com/artifact/org.modelmapper/modelmapper

Reply:
I mean we need to because if not it won't build

Reply:
Code changes are made. Can you take a look?